Travel Scheduler/Helper Files/TravelSchedulerHelper.h

```diff
@@ -9,6 +9,8 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+@class Place;
```

Comment:
Is that another way to import place.h?

Reply:
I think this is the correct way actually? (Idk, read it in the book and xCode errors if I put it in the import because both classes are importing each other and it causes errors).